{"name": "Проверка структуры сообщений об ошибках", "status": "passed", "start": 1760499365835, "stop": 1760499365835, "uuid": "9fdb993e-06fc-4951-b132-9631335bc285", "historyId": "b414fb0e3376a0bbe20849489b12a1f4", "testCaseId": "b414fb0e3376a0bbe20849489b12a1f4", "fullName": "test_modules.TestErrorMessages#test_error_messages_structure", "labels": [{"name": "suite", "value": "test_modules"}, {"name": "subSuite", "value": "TestErrorMessages"}, {"name": "host", "value": "DESKTOP-6FU1OQK"}, {"name": "thread", "value": "21764-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_modules"}], "titlePath": ["test_modules.py", "TestErrorMessages"]}